src/test/cypress/e2e/exam/test-exam/TestExamStudentExams.cy.ts

```diff
@@ -0,0 +1,147 @@
+import { Course } from 'app/entities/course.model';
+import { Exam } from 'app/entities/exam.model';
+import { CypressUserManagement, admin, studentOne, studentThree, studentTwo } from '../../../support/users';
+import {
+    courseManagement,
+    courseManagementAPIRequest,
+    examAPIRequests,
+    examExerciseGroupCreation,
+    examManagement,
+    examNavigation,
+    examParticipation,
+    navigationBar,
+    studentExamManagement,
+} from '../../../support/artemis';
+import { convertModelAfterMultiPart, generateUUID } from '../../../support/utils';
+import { CypressCredentials } from '../../../support/users';
+import { Exercise, ExerciseType } from '../../../support/constants';
+import dayjs from 'dayjs/esm';
+
+// Common primitives
+const uid = generateUUID();
+const examTitle = 'test-exam' + uid;
+const textFixture = 'loremIpsum-short.txt';
+const students: Array<CypressCredentials> = [studentOne, studentTwo, studentThree];
+const studentNames: string[] = [];
+
+const userManagement = new CypressUserManagement();
+
+let examExercise: Exercise;
+
+describe('Test Exam - student exams', () => {
+    let course: Course;
+    let exam: Exam;
+
+    before('Create course and exam', () => {
+        cy.login(admin);
+        courseManagementAPIRequest.createCourse().then((response) => {
+            course = convertModelAfterMultiPart(response);
+
+            const examConfig: Exam = {
+                course,
+                title: examTitle,
+                testExam: true,
+                startDate: dayjs().subtract(1, 'day'),
+                visibleDate: dayjs().subtract(2, 'days'),
+                workingTime: 5,
+                examMaxPoints: 10,
+                numberOfCorrectionRoundsInExam: 1,
+            };
+
+            for (const student of students) {
+                courseManagementAPIRequest.addStudentToCourse(course, student);
+            }
```

Comment:
Please move the examConfig directly next to the createExam()-call: 
```suggestion
            for (const student of students) {
                courseManagementAPIRequest.addStudentToCourse(course, student);
            }

            const examConfig: Exam = {
                course,
                title: examTitle,
                testExam: true,
                startDate: dayjs().subtract(1, 'day'),
                visibleDate: dayjs().subtract(2, 'days'),
                workingTime: 5,
                examMaxPoints: 10,
                numberOfCorrectionRoundsInExam: 1,
            };
```